src_js/components/main_content/enhanced_code_blocks/__tests__/useEnhancedCodeBlocks.test.ts

```diff
@@ -17,7 +17,27 @@ $ tar -xvzf starter_files.tar.gz
 </code></pre></div></div>`;
 const PLAINTEX_BLOCK_NUM_LINES = 4;
 
+const PYTHON_BLOCK = `<div class="lanuage-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code>"""This is a docstring."""
+      import math
+
+
+      def sqrt(x):
+        """Print the square root of x."""
+        print(math.sqrt(x))
+
+
+      if __name__ == "__main__":
+        print(sqrt(81))</code></div></div>`;
+
 describe('useEnhancedCodeBlocks', () => {
+  function triggerMouseEvent(node: HTMLElement | null, eventType: string) {
+    if (!node) {
+      throw new Error('node cannot be null');
+    }
+    const mouseEvent = new CustomEvent(eventType);
```

Comment:
I changed this to use `CustomEvent` instead of `document.createEvent` because the latter relies on `Event.initEvent`, which is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent).